azext_iot/tests/iothub/core/test_iot_ext_int.py

```diff
@@ -101,7 +130,7 @@ def tearDown(self):
     def test_storage(self):
         device_count = 1
 
-        content_path = os.path.join(CWD, "test_generic_replace.json")
+        content_path = os.path.join(CWD, "../test_generic_replace.json")
```

Comment:
Instead of the `../`, something like this is more env agnostic `os.path.join(os.path.dirname(CWD), "test_generic_replace.json")`